src/group_gui.cpp

```diff
@@ -281,19 +281,17 @@ class VehicleGroupWindow : public BaseVehicleListWindow {
 
 	/**
 	 * Draw a row in the group list.
-	 * @param y Top of the row.
-	 * @param left Left of the row.
-	 * @param right Right of the row.
+	 * @param r Rect to draw row in.
 	 * @param g_id Group to list.
 	 * @param indent Indentation level.
 	 * @param protection Whether autoreplace protection is set.
 	 * @param has_children Whether the group has children and should have a fold / unfold button.
 	 */
-	void DrawGroupInfo(int y, int left, int right, GroupID g_id, uint16_t level_mask = 0, uint8_t indent = 0, bool protection = false, bool has_children = false) const
+	void DrawGroupInfo(Rect r, GroupID g_id, uint16_t level_mask = 0, uint8_t indent = 0, bool protection = false, bool has_children = false) const
```

Comment:
const Rect& ?

Reply:
This time it gets modified within the function. Which is rare enough for it to perhaps feel like a mistake.